edb/lang/schema/_graphql.eql

```diff
@@ -19,7 +19,7 @@
 
 CREATE FUNCTION graphql::short_name(std::str) -> std::str
     FROM SQL $$
-        SELECT regexp_replace($1, '.+?::(.+$)', '\1');
+        SELECT regexp_replace($1, '.+?::(.+$)', '\1') || 'Type';
```

Comment:
Does this have to be an SQL function? Can you rewrite it to EdgeQL? (we'll disallow using SQL functions everywhere outside of `std.eql` soon).

Reply:
OK, I'll need to expose `regexp_replace` in `std` first.